fmn/rules/taskotron.py

```diff
@@ -29,8 +38,14 @@ def taskotron_task(config, message, task=None):
     if not task:
         return False
 
-    tasks = [item.strip().lower() for item in task.split(',')]
-    return message['msg']['task'].get('name').lower() in tasks
+    name = message['msg']['task'].get('name')
```

Comment:
Is there a reason that you got rid of `.lower()` here? That seems like something that could break existing rules if folks are using any upppercase letters in their settings